config/locales/codelists/BEIS/activity.en.yml

```diff
@@ -0,0 +1,8 @@
+---
+en:
+  covid19_related:
+    "0": Not related
+    "1": New activity full focus on COVID-19
+    "2": Existing activity adapted to fully focus on COVID-19
+    "3": New activity that will somewhat focus on COVID-19
+    "4": Existing activity adapted to somewhat focus on COVID-19
```

Comment:
Can we not use the Codelist introduced here somehow? They seem largely the same. https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/736/commits/9fec4002a851591798a2d1e193e6514fbd22750c

Reply:
Might be a point to share with the rest of the team - there's lots of duplication between codelists and I18n which might be unnecessary.

@CristinaRO This probably belongs in the main `activity.en.yml`, for consistency.

Reply:
> Can we not use the Codelist introduced here somehow? They seem largely the same.

@pezholio The pattern seems to be to keep "vendor data" such as codelists separate from "interface data" such as translation strings.

I interpreted this to mean that the "vendor" data is assumed to be a rigid set that we should keep intact so we can assimilate any changes to it without having to redo our tweaks, whereas we could choose (or have to) display the interface data in some other ways, with some edits.

> This probably belongs in the main activity.en.yml, for consistency.

@leeky I would counter that, for consistency with translations of the IATI codelists, it belongs in its own file in a BEIS structure. I didn't just slap a new file in, I thought about it and checked how it was done for the other codelists. If in fact there is no consistent pattern in the app, I propose we make a separate card for that, where we can compare and choose an approach that seems the most consistent at this point in time.

Reply:
@CristinaRO My mistake, I've been doing so much work with model transations and forgot that the IATI code list version existed (I know you didn't just "slap a new file in") :-/

I note that that iati.en.yml contains translations that aren't part of the standard, so we should migrate these over at some point.

Reply:
BTW, I'm keen to add these enhancements in the current sprint (in general, I know this is the last day of the current one). I have the experience of "refactoring sprints" being perpetually moved to the future, and I don't want to repeat it on this project! I just (you know, "just") don't want them to muddle the scope of the current PR.